meetings/bordeaux2018/index.md

```diff
@@ -24,23 +24,56 @@ The JuMP-dev workshop is a great opportunity to meet other people working on and
 
 ## Location
 
-The workshop will be held at the Institut de Mathématiques de Bordeaux, University of Bordeaux.
+The workshop will be held at the [Institut de Mathématiques de Bordeaux, University of Bordeaux](https://goo.gl/maps/PaeJbcRC58K2). If you are not staying within walking distance, the Talence campus is a short tram ride from the city centre.
+
 
 A variety of information about Bordeaux, as well as how to get there, and possible accommodation options, is available on the [ISMP conference website](https://ismp2018.sciencesconf.org).
 
-## Call for participants and talks
+## Call for participants
 
 The workshop is open for anybody to attend. In particular, we invite new contributors and those who have not met the core development team.
 
-Participants are encouraged to give a short (~20-minute) talk. The committee is looking for talks that will start off discussions during the workshop and beyond and for talks that present developments that the community may not be aware of. Talks will optionally be recorded and posted for those who cannot attend. We encourage speakers to give direct feedback on JuMP. See [last year's talks](https://www.youtube.com/watch?v=esOe5saQRKY&list=PLzK_rUGmc3o6EwPOCUCvBAbMJeYBS8PyY) for a flavor of the workshop.
-
-**[Use this form to indicate your participation and submit talk proposals](https://goo.gl/forms/tUeBUY6uAnAo8v5m2)**
-by **April 7th ~~March 15th~~**. There is no penalty for late sign-ups, but we may not be able to accommodate late talk proposals.
+The call for talks has now closed. However, if you wish to attend without giving a talk, **[use this form to indicate your participation](https://goo.gl/forms/tUeBUY6uAnAo8v5m2)**.
```

Comment:
Could you revert the changes to the form? We're still waiting for a couple abstracts and I had asked authors to use that form (I should've mentioned this). We can say in the form description that the call for talks is closed.

Reply:
I have reverted the form.